src/app/machines/components/MachineForms/MachineActionFormWrapper/MachineActionFormWrapper.tsx

```diff
@@ -184,6 +184,11 @@ export const MachineActionForm = ({
             {...commonNodeFormProps}
           />
         );
+      // No form should be opened for this, as it should only
+      // be available for machine details, and will be dispatched
+      // immediately on click.
+      case NodeActions.CHECK_POWER:
```

Comment:
Shouldn't that be the case for `NodeActions.LOCK` and `NodeActions.UNLOCK` too?

Reply:
On a machine's details page, yes, but not on the machine list. Unless we want to change that